app/src/main/java/com/infomaniak/mail/ui/main/menu/MenuDrawerFragment.kt

```diff
@@ -230,6 +231,7 @@ class MenuDrawerFragment : Fragment() {
     fun onDrawerOpened() {
         canNavigate = true
         mainViewModel.forceRefreshMailboxes()
+        requireActivity().trackScreen(this::class.java.name, this::class.java.simpleName)
```

Comment:
Just Fragment.trackScreen() is not enough?

Reply:
Done.